flair/data.py

```diff
@@ -1062,7 +1062,7 @@ def is_context_set(self) -> bool:
     def get_labels(self, label_type: str = None):
 
         # TODO: crude hack - replace with something better
-        if label_type:
+        if label_type and label_type != "deprel":
```

Comment:
Without this change the `get_labels` function fails:

```python
from flair.datasets.conllu import CoNLLUDataset

dataset: CoNLLUDataset = CoNLLUDataset('universal_dependencies.conllu')  # same example file as in the comment above

print(dataset[0].get_labels("deprel"))
```

```
Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "/vol/fob-vol7/mi18/dobbersc/PyCharmProjects/flair_fork/flair/data.py", line 1068, in get_labels
    self.add_complex_label(label_type, label=SpanLabel(span, span.tag, span.score))
  File "/vol/fob-vol7/mi18/dobbersc/PyCharmProjects/flair_fork/flair/data.py", line 300, in add_complex_label
    if typename in self.annotation_layers and label in self.annotation_layers[typename]:
  File "/vol/fob-vol7/mi18/dobbersc/PyCharmProjects/flair_fork/flair/data.py", line 257, in __eq__
    and self.head.id_text == other.head.id_text \
AttributeError: 'SpanLabel' object has no attribute 'head'
```

My guess is that inside the `add_complex_label` function the created `deprel` relation labels are incorrectly overwritten with span labels. The reason behind this is that the tokens have an individual `deprel` annotation in addition to the sentence labels.

I'm not sure how to fix this properly, since I'm not familiar with the "crude hack" that is mentioned here. I would appreciate it if you, @alanakbik, could take a look at this.

Reply:
@dobbersc I think recording deprels as relation labels is a great idea! But the refactoring involving complex labels is not yet complete and is currently blocked by some other refactorings that need to happen first. Since we currently do not yet predict this type of extended dependency trees in Flair anyway, I would wait with recording the deps until the rest is sorted out.

Reply:
Thank you for the review. I've implemented the requested changes and hope that we can include the deps relation labels after the refactorings again.